desktop/cmp/button/AppMenuButton.ts

```diff
@@ -143,7 +143,7 @@ function buildMenuItems(props: AppMenuButtonProps) {
             omit: hideAdminItem,
             text: 'Admin',
             icon: Icon.wrench(),
-            actionFn: () => window.open('/admin')
+            actionFn: () => XH.openWindow('/admin', 'xhAdmin')
```

Comment:
If I have multiple hoist apps or environments open will this cause one app to open its admin window in the same tab as another app?